src/cpp/src/continuous_batching_impl.hpp

```diff
@@ -30,6 +31,14 @@ class ContinuousBatchingPipeline::ContinuousBatchingImpl : public ContinuousBatc
     // flag to enable validation mode for sampler
     bool m_is_validation_mode_enabled = false;
 
+    // dynamic kv-cache allocation params
```

Comment:
looks like (at least) some of these params below are not used? as logic has been migrated to scheduler